src/index.js

```diff
@@ -1,4 +1,4 @@
-// /src/index.js - entry point
+// /src/index.js - entry point - minor change
 
 import './styles.css';
 import UI from './modules/ui';
```

Comment:
Nice work with the code :+1: 

I like the way you commented you work for readabity